packages/asset-kit-block/src/AssetKitBlock.tsx

```diff
@@ -44,7 +44,12 @@ export const AssetKitBlock = ({ appBridge }: BlockProps): ReactElement => {
         buttonText,
     } = blockSettings;
 
+    const hasTitle = title && hasRichTextValue(title);
+    const hasDescription = description && hasRichTextValue(description);
+
     const currentAssets = blockAssets[ASSET_SETTINGS_ID] ?? [];
+    const hasAssets = currentAssets.length > 0;
+    const shouldDisplayComponent = isEditing || hasTitle || hasDescription || hasAssets;
```

Comment:
`hasTitle || hasDescription` should not be part of this condition, as far as I know if there are no assets uploaded, we don't want to display the asset-kit block.

Reply:
Well, tough one.

Let's keep it. 👍 

Anyways, then you need to fix the lint errors in `AssetKitBlock.spec.ct.tsx`, you need to use single quotes instead of double. You can use the same approach I showed you on web-app.